packages/cyberstorm/src/components/Layout/CommunityListLayout/CommunityListSkeleton.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from "./CommunityListSkeleton.module.css";
+import { CommunityCardSkeleton } from "../../CommunityCard/ComunityCardSkeleton";
+
+export const CommunityListSkeleton = () => {
+  return (
+    <div className={styles.root}>
+      {[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].map((community) => (
```

Comment:
There's a `range` function in utils that could be used.

Reply:
Good point. I'll use it, Thanks!👌